ironfish-cli/src/commands/accounts/transactions.ts

```diff
@@ -16,6 +16,9 @@ export class TransactionsCommand extends IronfishCommand {
       char: 't',
       description: 'Transaction hash to get details for',
     }),
+    show: Flags.integer({
+      description: 'Number of latest transactions to get details for',
+    }),
```

Comment:
nit - I'm not sure that `show` is the best description of what this does. Maybe `limit` or `number`?